packages/table/src/table.tsx

```diff
@@ -357,12 +394,15 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         newRowHeights = Utils.assignSparseValues(newRowHeights, rowHeights);
 
         const newSelectedRegions = (selectedRegions == null) ? this.state.selectedRegions : selectedRegions;
+        const newFocusedCellCoordinates =
+            (focusedCellCoordinates == null) ? this.state.focusedCellCoordinates : focusedCellCoordinates;
```

Comment:
nit: when ternaries exceed the max line length, I tend to prefer:

```
const newFocusedCellCoordinates = (focusedCellCoordinates == null)
    ? this.state.focusedCellCoordinates
    : focusedCellCoordinates;
```

so you can read both cases easily.